src/protocol/subscriptions.cpp

```diff
@@ -31,73 +30,6 @@ namespace OpcUa
   }
 
   ////////////////////////////////////////////////////////
-  // CreateSubscriptionRequest
-  ////////////////////////////////////////////////////////
-
-  CreateSubscriptionRequest::CreateSubscriptionRequest()
-    : TypeId(CREATE_SUBSCRIPTION_REQUEST)
-  {
-  }
-
-  ////////////////////////////////////////////////////////
-  // DeleteSubscriptionRequest
-  ////////////////////////////////////////////////////////
-
-  DeleteSubscriptionRequest::DeleteSubscriptionRequest()
-    : TypeId(DELETE_SUBSCRIPTION_REQUEST)
-  {
-  }
-
-  DeleteSubscriptionResponse::DeleteSubscriptionResponse()
-    : TypeId(DELETE_SUBSCRIPTION_RESPONSE)
-  {
-  }
-
-
-  ////////////////////////////////////////////////////////
-  // SubscriptionData
-  ////////////////////////////////////////////////////////
-
-  SubscriptionData::SubscriptionData()
-    : RevisedPublishingInterval(100)
-    , RevisedLifetimeCount(30) //Should be 3 times keepalive
-    , RevizedMaxKeepAliveCount(10)
-  {
```

Comment:
It is just to help users doing stupid mistakes or spending a lot of time
setting things up

On Mon, Jun 8, 2015, 10:22 Denis Štogl <notifications@github.com> wrote:

> In src/protocol/subscriptions.cpp
> <https://github.com/FreeOpcUa/freeopcua/pull/147#discussion_r31894315>:
>
> > -
> > -  DeleteSubscriptionResponse::DeleteSubscriptionResponse()
> > -    : TypeId(DELETE_SUBSCRIPTION_RESPONSE)
> > -  {
> > -  }
> > -
> > -
> > -  ////////////////////////////////////////////////////////
> > -  // SubscriptionData
> > -  ////////////////////////////////////////////////////////
> > -
> > -  SubscriptionData::SubscriptionData()
> > -    : RevisedPublishingInterval(100)
> > -    , RevisedLifetimeCount(30) //Should be 3 times keepalive
> > -    , RevizedMaxKeepAliveCount(10)
> > -  {
>
> Ok I will put this back, even though subscription part works also without
> this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeOpcUa/freeopcua/pull/147/files#r31894315>.
>